### ISSUE TITLE
Create milestones of what needs to happen in a decent timelined fashion

### ISSUE THREAD

**Issue by Qkyrie** [FundRequest/crowdsale#12]
I'd like to prioritize tasks based on time based events.

for example:
urgent tasks which need to happen before my holidays, tasks which are necessary before the crowdsale, must haves, nice-to-haves etc..

**Comment by karelstriegel** [comment]
https://github.com/FundRequest/crowdsale/milestones
